### ISSUE TITLE
Any limits on grpc response size?

### ISSUE THREAD

**Issue by AlexKovalevych** [elixir-grpc/grpc#173]
**Describe the question**
I'm getting an error when grpc server tries to send a big response (~400 kb):

```
{"log":"Ranch listener \"FooBar.GRPC.Endpoint\" had connection process started with :cowboy_clear:start_link/4 at #PID<0.11017.0> exit with reason: {{:case_clause, {:trailers, [{\"grpc-message\", \"\"}, {\"grpc-status\", \"0\"}]}}, [{:cow_http2_machine, :queue_data, 4, [file: '/app/deps/cowlib/src/cow_http2_machine.erl', line: 1339]}, {:cow_http2_machine, :send_or_queue_data, 4, [file: '/app/deps/cowlib/src/cow_http2_machine.erl', line: 1199]}, {:cowboy_http2, :maybe_send_data, 4, [file: '/app/deps/cowboy/src/cowboy_http2.erl', line: 770]}, {:cowboy_http2, :commands, 3, [file: '/app/deps/cowboy/src/cowboy_http2.erl', line: 634]}, {:cowboy_http2, :loop, 2, [file: '/app/deps/cowboy/src/cowboy_http2.erl', line: 247]}, {:cowboy_http, :parse, 2, [file: '/app/deps/cowboy/src/cowboy_http.erl', line: 311]}, {:proc_lib, :init_p_do_apply, 3, [file: 'proc_lib.erl', line: 249]}]}\n","severity":"ERROR","sourceLocation":{"file":null,"line":0,"function":null},"time":"2020-09-16T12:41:46.655Z"}
```

**Versions:**
- OS: alpine 3.9
- Erlang: 22
- Elixir: 1.9.1
- mix.lock(grpc, gun, cowboy, cowlib):
grpc - 6493159b763530f7d04a904885ab315d2cafc05f
gun - 2.0.0
cowboy - 2.7.0
cowlib - 2.8.0

**Comment by ljmarks** [comment]
Seems related to this: https://github.com/ninenines/cowlib/issues/92 which should have been fixed in cowlib 2.9. 

I would try and update cowlib. Here's one way to do that:
Add this:
```elixir
 {:cowlib, "~> 2.9.0", override: true}
```
to the bottom of your deps list in `mix.exs` then do `mix deps.update cowlib`.

**Comment by AlexKovalevych** [comment]
yup, that was it. I had to stay on 2.8.0 cowlib since it couldn't resolve dependencies with 2.9.0 (unfortunately don't remember which library caused the problem), but now it compiles and works fine, thank you!
